gateway/modules/crud/mgo/read.go

```diff
@@ -24,6 +24,8 @@ func (m *Mongo) Read(ctx context.Context, col string, req *model.ReadRequest) (i
 	}
 	collection := m.getClient().Database(m.dbName).Collection(col)
 
+	req.Find = sanitizeWhereClause(ctx, col, req.Find)
```

Comment:
Are you doing this for update and delete as well?

Reply:
Nope, I'll add it